File: assets/rest/samples/javascript/AccountsAPI.getAccount.get.js
import { rise } from 'risejs'

rise.accounts
  .getAccount('8093718274007724701R')
  .then(function({ account }) {
    console.log(account.balance) // 2973803650603
  })
  .catch(function(err) {
    console.log('Error: ', err) // handle error
  })

File: assets/rest/samples/elixir/AccountsAPI.getAccount.get.exs
import RiseElixir

try do
  result = RiseElixir.Accounts.get_account(%{address: "8093718274007724701R"})
  IO.puts("Balance: #{result["account"]["balance"]} RISE")
  # Balance: 3370390005074 RISE
catch
  err -> "API error #{err}"
end
